web/servlet/NavBar_cs.html

```diff
@@ -64,6 +64,13 @@
                         <li><a href="/tables/type">Všechny Typy</a></li>                        
                     </ul>
                 </li>
+                <li class="dropdown context-permissions">
+                    <a href="/permission" class="dropdown-toggle" data-toggle="dropdown">Permission <b class="caret"></b></a>
+                    <ul class="dropdown-menu">
+                        <li><a href="/permission/login">Login</a></li>
+                        <li><a href="/permission/logout">Log out</a></li>
```

Comment:
Perhaps I don't understand the syntax here, but I don't see a Logout.shtml file to match the Login.html file.

Reply:
Thanks for the review!

Login and logout is handled by the [PermissionServlet.java](https://github.com/JMRI/JMRI/pull/13484/files#diff-c13e48a73a9a34cbf21e096908ce43cd03388878cdbca53fa29d11c0b076b0fd). The request is read on line 31 and handled by the switch-case statement on lines 34-46.

The [Login.html](https://github.com/JMRI/JMRI/pull/13484/files#diff-8ce8ab46d2ca7c637f5ac31be38d762774302261b1f78d0f0b89dba38f65c92b) page has a html form where the user enters the username and password.

When the user logout, a page with a simple message is shown telling the user that he has logged out. Therefore I have a generic [Response.html](https://github.com/JMRI/JMRI/pull/13484/files#diff-fe216ea126fe6085df23525e85c5cd3c07af31351dbdc113b30d3a47e00b3dff) page for several different messages, like `Login was successful`, `Logout was successful`, `Bad username or password` and `Permission denied`.